ts/util/version.ts

```diff
@@ -1,44 +1,19 @@
-import fs from 'fs'
-import path from 'path'
-
-/**
- * Lazily read, cached, package version
- */
-let VERSION: string
+import { version as pkgVersion } from '../../package.json'
```

Comment:
Yep, this is obvious, more elegant way to do this. But I have been caught in the past by a gotcha that this creates related to having a nested `package.json` in dist. IIRC, it only rears its head in dependants of this package. I'll check and report back.

Reply:
Checked, and this won't be a problem because no `package.json` ends up in `dist` in any case